codenav/src/codenavigation.c

```diff
@@ -261,7 +261,8 @@ on_configure_response(GtkDialog* dialog, gint response, gpointer user_data)
 									
 			/* If one field is empty, ignore this line (it will be replaces
 			   at next execution) */ 
-			if ( strlen(impl_list[i])==0 || strlen(head_list[i])==0 )
+			if ( !impl_list[i] || strlen(impl_list[i])==0 ||
```

Comment:
BTW, it's not this patche's fault, and many compilers are likely gonna optimize some cases, but it's a bit silly to use `strlen(x) == 0`, `! *x` is gonna be quite a tad faster.